bundles/org.openhab.binding.miio/src/main/resources/ESH-INF/thing/vacuumThing.xml

```diff
@@ -294,7 +294,7 @@
 		<state pattern="%.0f" readOnly="true" />
 	</channel-type>
 	<channel-type id="last_clean_finish">
-		<item-type>Boolean</item-type>
+		<item-type>Number</item-type>
```

Comment:
Are you sure `Number` is the correct item-type? `Switch` would be closer to `Boolean` but I didn't look in the code to check what is actually send to that channel.